app/views/layouts/application.html.erb

```diff
@@ -23,9 +23,19 @@
 
   <!-- Stylesheets -->
   <%= stylesheet_link_tag 'application', media: 'all' %>
+  <% if !@extra_css.blank? %>
+    <% @extra_css.each do |css| %>
+  <%= css %>
+    <% end %>
+  <% end %>
 
   <!-- Scripts -->
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => false %>
+  <% if !@extra_js.blank? %>
+    <% @extra_js.each do |js| %>
+  <%= js %>
```

Comment:
can you make sure that this and the above <%= css %> tag are indented far enough?  small complaint, I know :)